src/plotting/plotSettings/moreOptions/moreOptionsBox.component.test.tsx

```diff
@@ -122,4 +128,27 @@ describe('MoreOptionsBox', () => {
 
     expect(changeSelectedPlotChannels).toHaveBeenLastCalledWith(expected);
   });
+
+  it('allows user to switch channel axis', async () => {
+    const expected = testPlotDatasets.map((dataset, i) => ({
+      name: dataset.name,
+      options: {
+        visible: true,
+        colour: `colour-${i.toString()}`,
+        lineStyle: 'solid',
+        yAxis: i === 1 ? 'right' : 'left',
+      },
+    }));
+
+    createView();
+
+    await user.click(
+      screen.getByLabelText(`Pick ${props.channel.name} colour`)
+    );
```

Comment:
Old copy-pasted code?

Reply:
Oops - good spot! Yeah I just copied the previous test as the base of this one 😅